maxine/python/euler/problem7.py

```diff
@@ -0,0 +1,12 @@
+'By listing the first six prime numbers: 2, 3, 5, 7, 11, and 13, we can see that the 6th prime is 13. What is the 10 001st prime number?'
+
+def is_prime(n):
+    for d in range (2,n):
+        if n%d ==0:
+            return False
+    return True
+
+def nth_prime(n):
+    
+a = list(i for i in range(2,1e9) if is_prime(i))
+a[101]
```

Comment:
Why 101?

Reply:
silly mistake, this was supposed to be 10001